libs/cdk/src/lib/render-strategies/model.ts

```diff
@@ -0,0 +1,37 @@
+import { Notification, Observable } from 'rxjs';
+import { ChangeDetectorRef } from '@angular/core';
+
+export enum RxNotificationKind {
+  suspense = 'suspense',
+  next = 'next',
+  error = 'error',
+  complete = 'complete'
+}
+
+type NotificationExtract = 'value' | 'hasValue';
```

Comment:
```suggestion
type NotificationPick = 'value' | 'hasValue';
```
or maybe `NotificationSlice`